### PR TITLE
[dpg] update spread doc

### PR DESCRIPTION
**Pull request by tadelesh**
doc for the new behavior of spread, including one sample to show impact for azure core template.

### CODE REVIEW

File: docs/howtos/DataPlane Generation - DPG/08methodInputs.mdx [hunk 1]
```diff
@@ -773,3 +775,171 @@ public void register(Schema schema);
...
+```python
+class RepairInfo:
+  problem: str
+  contact: str
+
+class RepairResult:
+  reason: str
+  info: str
+
+def scheduleRepairs(self, problem: str, contact: str, *, content_type: str = "application/json") -> RepairResult:
```
Comment (weidongxu-microsoft) [inline docs/howtos/DataPlane Generation - DPG/08methodInputs.mdx:825]: You seems missed the `widgetName`

https://cadlplayground.z22.web.core.windows.net/cadl-azure/?c=aW1wb3J0ICJAdHlwZXNwZWMvcmVzdCI7CskZYXp1cmUtdG9vbHMvyCUtxhVjb3JlIjsKCnVzaW5nIFR5cGVTcGVjLkh0dHA70BVSZXN00RVWZXJzaW9uaW5nyBtBxFkuQ29yZTsKCkBzZXJ2aWNlKHsKICB0aXRsZTogIkFwcCIsCn0pCkB2xj5lZChBcGnHS3MpCm5hbWVzcGFjZSBDYWRsLkFwcDsKCmVudW0gyycgxFNAdXNlRGVwZW5kZW5jeShnbG9iYWwu6gCD6ACgcy52MV8wX1ByZXZpZXdfMikKICB2MjAyMl8wNl8wMV9wxhk6ICLEFS0wNi0wMS3HFeQArAoKQHJlc291cmNlKCJ3aWRnZXRzIikKbW9kZWwgV8UQ5gCQa2V5yCFOYW1lIsRp5ADPOiBzdHLlASZ9xzZSZXBhaXJJbmZvxTpwcm9ibGVtyiggIGNvbnRhY3TYO1Jlc3VsxndyZWFzb27MPGluZm%2FMOWFsaWFzIE9wZXJhdOUBPj3rAankAdvlANzKIDx7fT47Cm9wIHNjaGVkdWxlxnRzIGnMRsk5QWPEEDzmAQos6wDcyAzmAK0%2BOwo%3D&e=%40azure-tools%2Ftypespec-autorest&options=%7B%7D

**Review by iscai-msft — APPROVED** [review]

**Comment by tadelesh** [comment]
@ArcturusZhang @archerzz could you help to update the .net part?
@qiaozha could you help to update the js part?

**Comment by qiaozha** [comment]
> @ArcturusZhang @archerzz could you help to update the .net part? @qiaozha could you help to update the js part?

JS part is done.

**Comment by ArcturusZhang** [comment]
@tadelesh C# part updated

**Comment by tadelesh** [comment]
Thank you all.

**Pull request merged**